src/main/java/com/ecom/shopping/adminkart/controller/ProductCategoryController.java

```diff
@@ -14,6 +29,10 @@
 public class ProductCategoryController {
     @Autowired
     private ProductCategoryService productCategoryService;
+    @PostMapping("/addCategory")
+    public void addCategory(@RequestBody ProductCategoryDto productCategoryDto) {
```

Comment:
why return type is void for this method. it should be proper response like ProductCategoryDto